schema.sql

```diff
@@ -58,4 +58,18 @@ CREATE TABLE visits (
     vet_id INT REFERENCES vets (id) ON DELETE RESTRICT ON UPDATE CASCADE,
     animal_id INT REFERENCES animals (id) ON DELETE RESTRICT ON UPDATE CASCADE,
     date_of_visits DATE
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
As suggested by the previous reviewer, since these queries add data to the database, should be placed in the `data.sql` file. Only the queries which modify the structure of the tables should be in the `schema.sql` file. Kindly move this code into the `data.sql` file.